velox/functions/prestosql/tests/TypeOfTest.cpp

```diff
@@ -69,6 +70,19 @@ TEST_F(TypeOfTest, basic) {
   EXPECT_EQ("qdigest(real)", typeOf(QDIGEST(REAL())));
   EXPECT_EQ("qdigest(double)", typeOf(QDIGEST(DOUBLE())));
 
+  auto enumName = "test.enum.mood";
```

Comment:
This test is very long. Would you move the new logic into a separate test method?